pages/api/address/stats.ts

```diff
@@ -84,9 +85,7 @@ export default withApiHandler(async (req: NextApiRequest, res: NextApiResponse)
       : (req.query.address as string[]);
 
   if (!req.query.address) {
-    return res.status(400).json({
-      error: 'Missing address'
-    });
+    throw new ApiError('Address stats, missing address', 401, 'Missing address');
```

Comment:
401 is typically used for `Unauthorized` where 400 is for `bad request`. Wouldn't 400 make more sense here?

Reply:
yes! typo